utilities/__init__.py

```diff
@@ -0,0 +1,4 @@
+import utilities_setup

```

Comment:
It is a bit redundant to name the files in the utilities folder `utilities_<name>`. Can we just go with `<name>`? What do you think?

Reply:
good suggestion. I changed the file name